### ISSUE TITLE
Apex 3 Keyboard locks up when running apexctl, have to reseat cable.

### ISSUE THREAD

**Issue by bonkmaykr** [AstroSnail/apexctl#11]
The title says it all:
https://youtu.be/W81LvtMsQ0w

**Comment by AstroSnail** [comment]
Thanks for the report, I guess I learned that I need to be careful when publishing code for devices I can't test.

Unfortunately, the very fact that I don't have an Apex 3 to test with means I can't realistically support it. I have no idea what's going on, and no clue how to fix it, and nobody to help me figure it out.

I've moved the Apex 3 and Apex M800 support behind an opt-in make flag, so you can try playing with it if you want to, while hopefully saving others from this annoying situation.

**Comment by bonkmaykr** [comment]
I could always make a donation. I got the keyboard for only like 50 USD

**Comment by AstroSnail** [comment]
I would *very much* appreciate that, but I should mention I don't live in the United States, so we'll have to work out shipping.

**Comment by AngelQC** [comment]
@AstroSnail any development toward the support of Apex 3?
Given that it locks up the keyboard, I am not too keen on trying it :)

**Comment by gardotd426** [comment]
Yeah thank god I actually looked before running this on my brand new Apex 3, after trying several other github steelseries kb linux utilities blindly and none of them really working lol

**Comment by MikyVlc** [comment]
> Yeah thank god I actually looked before running this on my brand new Apex 3, after trying several other github steelseries kb linux utilities blindly and none of them really working lol

OpenRGB can control Apex 3 without problem, you must create a profile because save function is not available.

Greetings
